src/server/address_manager.py

```diff
@@ -507,13 +507,13 @@ def resolve_tried_collisions_(self):
                         resolved = True
                     elif time.time() - old_info.last_try < 4 * 60 * 60:
                         if time.time() - old_info.last_try > 60:
-                            self.mark_good_(peer, False, time.time())
+                            self.mark_good_(peer, False, math.floor(time.time()))
```

Comment:
I'm not 100% sure about the correctness of this, but was used in other place like this.

Reply:
Its fine